<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>userRegWindow</class>
 <widget class="QMainWindow" name="userRegWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>421</width>
      <height>531</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="nameLine">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>321</width>
       <height>25</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Type your name</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>256</width>
       <height>81</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;USER REGISTRATION&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>480</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Finish</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>480</y>
       <width>231</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="cpfLine">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>180</y>
       <width>321</width>
       <height>25</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Type your cpf</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="birthLine">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>230</y>
       <width>321</width>
       <height>25</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Type your birth</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="emailLine">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>280</y>
       <width>321</width>
       <height>25</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Type your email</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="passwordLine">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>330</y>
       <width>321</width>
       <height>25</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Type your password</string>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>390</y>
       <width>138</width>
       <height>54</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="commomButton">
        <property name="text">
         <string>Commom user</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="serviceButton">
        <property name="text">
         <string>Service Provider</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>639</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
